README.adoc

```diff
@@ -1,17 +1,223 @@
-= Android uLink
+= uProtocol Client Android Java Library
 :toc:
+:toclevels: 4
+:sectnums:
+:source-highlighter: coderay
 
 == Overview
+The following is the uProtocol client library that implements uTransport, RpcClient and RpcService defined in https://github.com/eclipse-uprotocol/up-java[uProtocol Java SKD] using Android Binder. It also includes some commonly used utilities for error handling.
 
-the following is the Java Android (Binder) ulink library that implements uTransport and RpcClient defined in https://github.com/eclipse-uprotocol/uprotocol-spec/tree/main[uProtocol Specifications]. 
+== Getting Started
+=== Importing the Library
+If you are using Gradle, add the following to your _build.gradle_ file's dependencies:
 
+[,groovy]
+----
+android {
+    dependencies {
+        implementation 'org.eclipse.uprotocol:up-client-android-java::1.5.+'
+    }
+}
+----
 
-== Getting Started
+=== Configuring the Library
+`ULink`, by default, establishes a connection to uBus service that is integrated into the system as part of `"org.eclipse.uprotocol.core"` package.
+
+If a service that implements `IUBus.aidl` interface is integrated in a different package, you should configure the library by specifying that component or just that package.
```

Comment:
Do developers need to ever care about IUbus.aidl? If not then I would keep out of of this README.

Reply:
They might be.

Reply:
I don't think so as the upclient will be tightly coupled to a up implementation so one would simple update up-client-android library to pull in the latest AIDL that would talk to the latest up-android etc...

Reply:
resolve for now, we can remove as needed later.